### PR TITLE
Allow auto_padding_min_aspect to be set in X:Y (e.g. 24:9) form

### PR DESCRIPTION
**Pull request by donaldguy**
This one is a little sillier, but seemed apropos while doing percentages (in #27). But I also rebased to buildably/reviewably seperate them so you could take one or the other, or both, as you like

This one _probably_ only makes sense with some tweaks to make sure a ~fine delta works to let people set _to_ rather than just below their relevant monitor(s)

Anyway:

```
$ yabai -m config auto_padding_min_aspect
20:9
$ yabai -m config auto_padding_min_aspect 24:9
$ yabai -m config auto_padding_min_aspect
24:9
$ yabai -m config auto_padding_min_aspect 2.333
$ yabai -m config auto_padding_min_aspect
2.3330
```

(and then the triggering or not ; I'm on a 32:9 right now and can confirm it turns off autopad behaviors at `33:9` (and sadly 32:9), and comes back once I put to `31:9`)

### CODE REVIEW

**Comment by FelixKratz** [comment]
Thanks for all of the PRs, I have squashed them into a single patch for the autopadding feature: https://github.com/FelixKratz/yabai/commit/a53fd3b049002648daf1782afff00406d9d4ff8b

**Comment by donaldguy** [comment]
👍

As I said in description, this one probably begs for some follow-up to let the ratios _match_ rather than come in under?

(or just fudge it and have the calculation subtract a little (1%?) when using the X:Y form?)

**Comment by FelixKratz** [comment]
Its probably because some aspect ratios are not expressed accurately enough such that a fine delta would probably be adequate.

File: src/space_manager.c [hunk 1]
```diff
@@ -1171,10 +1177,9 @@ void space_manager_begin(struct space_manager *sm)
     // this is just to avoid awkward sm->autopad.x access everywhere :/
     sm->autopad = &sm->_autopad;
     sm->autopad->enabled = false;
-    sm->autopad->min_aspect = 20. / 9.;
+    space_manager_set_autopad_min_aspect(sm, 20., 9);
     sm->autopad->width_type = SPACE_AUTOPAD_VALUE_FIXED_INT;
     sm->autopad->width = 840;
-    sm->autopad->height_type = SPACE_AUTOPAD_VALUE_FIXED_INT;
```
Comment (donaldguy) [inline src/space_manager.c:1177]: While I was overcurating the trunk on my fork to re-emphasize/correctly-date my meager contributions (For an audience of exactly no one; and which I really hope hasn't been spamming you per the "commit mentioned PR" messages)... 

I realized this line got errantly dropped in last rebase of this on #27 

I haven't tested and its definitely possible that under clang the order of the enum makes this be the implicit default

but its also possible it'll cause an erroneous fall-through to else here:

https://github.com/FelixKratz/yabai/blob/1dd1d023e5a107961b8246e6d0298d357e2c983a/src/space_manager.c#L514-L521)

iff in fact someone's yabairc specifies `-m config auto_padding on` but nothing for `auto_padding_height`

**Pull request merged**